.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -44,10 +44,7 @@ body:
       label: Desktop Platform Information
       description: |
         Would you mind to tell us the system information about your desktop platform?
-      value: |
-        - OS: [e.g. iOS]
-        - Browser [e.g. chrome, safari]
-        - Version [e.g. 22]
+      placeholder: "example: MacOS 12.2 Logseq Desktop v0.5.9"
```

Comment:
Better use "OS version, Browser or native App, Logseq App version"

Reply:
Do you mean
```suggestion
      placeholder: |
        OS version, Browser or App, Logseq App version
        example: MacOS 12.2, App, v0.5.9
```